sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/TypeScriptRules.java

```diff
@@ -139,12 +139,13 @@
   public static final Set<String> FILE_LEVEL_RULES = Collections.unmodifiableSet(new HashSet<>(Arrays.asList("S113", "S104", "S1451")));
 
   private static final String RESOURCE_FOLDER = "org/sonar/l10n/typescript/rules/typescript";
+  private static final String DEFAULT_PROFILE_PATH = RESOURCE_FOLDER + "/" + "Sonar_way_profile.json";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this hard-coded path-delimiter. ([squid:S1075](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1075))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=sonarlint-integration-poc&issues=AWD5K7rZEFfjxkQcPcLA&open=AWD5K7rZEFfjxkQcPcLA)

[ISSUE_KEY]: AWD5K7rZEFfjxkQcPcLA